baseplate/frameworks/queue_consumer.py

```diff
@@ -145,7 +176,7 @@ def get_batch(self, max_items, timeout):
         return batch
 
 
-class KombuConsumer(BaseKombuConsumer):
+class KombuConsumer:
```

Comment:
Broke up inheritance here and opted for composition instead. The signatures of the `get_` methods are different so claiming it's a subclass is a lie.